src/Illuminate/Mail/TransportManager.php

```diff
@@ -97,8 +97,8 @@ protected function createMailDriver()
      */
     protected function createMailgunDriver()
     {
-        $client = new HttpClient;
         $config = $this->app['config']->get('services.mailgun', []);
+        $client = new HttpClient(Arr::get($config, 'guzzleOptions', []));
```

Comment:
Adding ``use Illuminate\Support\Arr;`` in the file ``Illuminate\Routing\Router`` sets all the tests to green. I don't if this is a clean way tho ;)